youtube_dl/extractor/gaskrank.py

```diff
@@ -33,10 +55,30 @@ def fix_json(code):
         categories = [re.match(self._VALID_URL, url).group('categories')]
         title = self._search_regex(r'movieName\s*:\s*\'([^\']*)\'', webpage, 'title')
         thumbnail = self._search_regex(r'poster\s*:\s*\'([^\']*)\'', webpage, 'thumbnail', default=None)
+
+        mobj = re.search(r'Video von:\s*(?P<uploader_id>[^|]*?)\s*\|\s*vom:\s*(?P<upload_date_day>[0-9][0-9]).(?P<upload_date_month>[0-9][0-9]).(?P<upload_date_year>[0-9][0-9][0-9][0-9])', webpage)
+        uploader_id = mobj.groupdict().get('uploader_id')
+        upload_date = mobj.groupdict().get('upload_date_year') + mobj.groupdict().get('upload_date_month') + mobj.groupdict().get('upload_date_day')
+        if len(upload_date) != 8:
+            upload_date = None
+
+        uploader_url = self._search_regex(r'Homepage:\s*<[^>]*?>(?P<uploader_url>[^<]*)', webpage, 'uploader_url', default=None)
+        tags = re.findall(r'/tv/tags/[^/]*?/\"\s*>(?P<tag>[^<]*?)<', webpage)
+
+        view_count = self._search_regex(r'class\s*=\s*\"gkRight\"(?:[^>]*>\s*<[^>]*)*icon-eye-open(?:[^>]*>\s*<[^>]*)*>\s*(?P<view_count>[0-9\.]*)', webpage, 'view_count')
+        if view_count:
+            view_count = int(view_count.replace('.', ''))
+
+        average_rating = self._search_regex(r'itemprop\s*=\s*\"ratingValue\"\s*>\s*(?P<average_rating>[0-9,]*)', webpage, 'average_rating')
```

Comment:
Fixed both.
Also, removed all unnecessary escaped double quotes.
Also, removed the tags from the tests because they might change on an already uploaded video.